src/Umbraco.Web.UI.Client/src/views/components/umb-media-grid.html

```diff
@@ -19,26 +21,79 @@
+                               extension="{{item.extension}}"
+                               icon="{{item.icon}}"
+                               size="s">
+                </umb-file-icon>
 
+            </div>
         </div>
     </div>
-</div>
+    <!-- markup for the media list -->
+
+    <div class="umb-media-grid__list-view" ng-show="showMediaList">
+            <div class="umb-table umb-table--condensed">
+                <div class="umb-table-head">
+                    <div class="umb-table-row">
+                        <div class="umb-table-cell">
+                            <input class="umb-table__input" type="checkbox"
+                                   ng-show="vm.allowSelectAll"
+                                   ng-click="vm.selectAll($event)"
+                                   ng-checked="vm.isSelectedAll()">
+                        </div>
+                        <div class="umb-table-cell  umb-table__name">
+                            <button type="button"
+                                    class="umb-table-head__link sortable"
+                                    ng-click="setSort('name')"
+                                    prevent-default>
+                                <localize key="general_name">Name</localize>
+                                <i class="umb-table-head__icon icon" aria-hidden="true" ng-class="{'icon-navigation-up': isSortDirection('name', 'asc'), 'icon-navigation-down': isSortDirection('name', 'desc')}"></i>
+                            </button>
+                        </div>
+                        <div class="umb-table-cell">
+                            <button type="button"
+                                    class="umb-table-head__link sortable"
+                                    ng-click="setSort('updateDate')"
+                                    prevent-default>
+                                Last Updated
+                                <i class="umb-table-head__icon icon" ng-class="{'icon-navigation-up': isSortDirection('updateDate', 'asc'), 'icon-navigation-down': isSortDirection('updateDate', 'desc')}"></i>
+                            </button>                           
+                        </div>
+               
+                    </div>
+                </div>
+                <div class="umb-table-body">
+                    <div class="umb-table-row umb-media-grid__list-item -selectable" ng-click="clickItem(item, $event, $index)" ng-repeat="item in items | filter:filterBy | orderBy:sortBy:sortReverse" ng-class="{'-selected': item.selected}">
+
+                        <div class="umb-table-cell">
+                            <i ng-show="item.selected" class="umb-table-body__icon umb-table-body__checkicon icon-check" aria-hidden="true"></i>
+                            <i class="{{item.icon}}" ng-if="!item.thumbnail && item.extension != 'svg' && !item.selected"></i><i class="icon-picture" ng-if="item.thumbnail && !item.selected"></i>
```

Comment:
Can add `aria-hidden="true"` here as well for both of them

Reply:
> Can add `aria-hidden="true"` here as well for both of them

Added (one was there already)